File: bindings/rust/denali-sys/examples/simple.rs
fn main() {
    unsafe {
        let display = denali_sys::dwl_display_connect(std::ptr::null_mut());
        if display.is_null() {
            eprintln!("Failed to connect to display");
            return;
        }
        println!("Connected!");

        denali_sys::dwl_display_disconnect(display);
    }
}

File: examples/simple.c
#include "dwl/display.h"

#include <stdio.h>

int main(void) {
    DwlDisplay* display = dwl_display_connect(nullptr);
    if (!display) {
        printf("Error connecting to display!\n");
        return -1;
    }

    printf("Connected!\n");

    dwl_display_disconnect(display);
}
